src/sfizz/Voice.cpp

```diff
@@ -718,3 +714,59 @@ void sfz::Voice::setupOscillatorUnison()
     }
 #endif
 }
+
+void sfz::Voice::prepareSmoothers(const ModifierArray<size_t>& numModifiers)
+{
+    for (auto& mod : allModifiers)
+        modifierSmoothers[mod].resize(numModifiers[mod]);
+}
+
+void sfz::Voice::pitchEnvelope(absl::Span<float> pitchSpan) noexcept
+{
+    const auto numFrames = pitchSpan.size();
+    auto bends = resources.bufferPool.getBuffer(numFrames);
+    if (!bends)
+        return;
+
+    const auto events = resources.midiState.getPitchEvents();
+    const auto bendLambda = [this](float bend) {
+        return region->getBendInCents(bend);
```

Comment:
@paulfd, I'm trying to rebase this.
This function name is misleading. It returns a factor not cents.